server/usecases/board/github.go

```diff
@@ -347,3 +355,52 @@ func githubLabelLists(issue *github.Issue, projectLists []*list.List) []list.Lis
 	}
 	return githubLabels
 }
+
+func InjectRedis() *redis.Client {
```

Comment:
__[Golint]__ exported function InjectRedis should have comment or be unexported ([link](https://sider.review/gh/repos/50192141/pulls/456#issue-3827455))

<sub>posted by Sider</sub>